doc/gallery_src/lightning-creek-magnetic.py

```diff
@@ -17,6 +17,11 @@
 Inlier, QLD (P1029), magnetic line data, AWAGS levelled. Geoscience Australia,
 Canberra <http://pid.geoscience.gov.au/dataset/ga/142419>`__
 
+**Pre-processing:**
+[Source code](https://github.com/fatiando-data/lightning-creek-magnetic-grid)
+that prepares the original dataset for redistribution by filtering,
+standardizing, converting coordinates, compressing etc.
```

Comment:
```suggestion
**Pre-processing:** `Source code for preparation of original dataset
for redistribution in Ensaio 
<https://github.com/fatiando-data/lightning-creek-magnetic-grid>`__
```

Reply:
This script is made into a gallery page by sphinx-gallery. Unfortunately, both sphinx-gallery scripts and docstrings have to use reStructuredText instead of Markdown. Copied over the same link from the docstring here.